### PR TITLE
Export the module and make it possible to pass option to the constructor

### PR DESCRIPTION
**Pull request by thomas-lebeau**
_PR in progress, need feedback._

This Expose the Zoomable class publicly. The goal is to be able to import it directly from another es6 project. Then we could just:

``` js
import Zoomable from 'minimal-lightbox';

new Zoomable(document.getElementById('my-image'), options);
```

Useful if we want for exemple to preload images or if we add images dynamically.
### The problems are:
- In order to import it like that, the `main` attribute in `package.json` need to point to `src/Zoomable.js` but  we then loose the ability to wiredep the compiled version for exemple. I do think we should make the move, it's more future proof, and you could just manually add the compiled version with script tags in your html, that does not repose on the `main` attribute.
- Also, I'm thinking that the options could **also** been read from `data-attributes`, that way you just have to add the script and it will be configured and don't need another javascript to have custom configuration.
- Finally, if I want to manually bind the zoom, I will not add `data-action=zoom` but then some styles use that as a selector. I think we should remove the reference to data-action in the css and replace it with a class that will be added by the constructor.

---

**tl;dr:** make it possible to use the plugin in 2 different ways:
- `new Zoomable(element, options)`

**or**  
- ``

_ref: #6_

### CODE REVIEW

**Comment by mshwery** [comment]
The data attrs part isn't a big change. That can happen on config initialization.

For `import Zoomable from 'minimal-lightbox';` we don't necessarily need to have it return es6 code. We _could_ expose the src file in `package.json` as an extra property. I don't think `main` should point to the src/Zoomable.js because of the ES6 transpilation dependency.

As long as Zoomable.js has an export and a UMD wrapper you should be able to `require('minimal-lightbox')` or `import Zoomable from 'minimal-lightbox';`

Babel has options to automatically wrap your output file with UMD: http://babeljs.io/docs/usage/modules/#umd

But right now browserify is producing the output. I can't recall if it automatically enables `require`ing the output file.

Maybe something like this? https://www.npmjs.com/package/umd-wrap or https://github.com/eduardolundgren/gulp-umd

Could add this process (and the current `compile` script) to both be part of the `prepublish` script event hook.

**Comment by mshwery** [comment]
Regarding the styles – perhaps Zoomable should apply its own styles to the element instead of relying on data attributes.
